spring-ai-core/src/main/java/org/springframework/ai/chat/client/advisor/api/StreamAroundAdvisor.java

```diff
@@ -31,11 +26,9 @@ public interface StreamAroundAdvisor extends Advisor {
 	/**
 	 * Around advice that wraps the invocation of the advised request.
 	 * @param advisedRequest
-	 * @param adviceContext
 	 * @param chain the chain of advisors to execute
 	 * @return the result of the advised request
 	 */
-	Flux<ChatResponse> aroundStream(AdvisedRequest advisedRequest, Map<String, Object> adviceContext,
-			AroundAdvisorChain chain);
+	Flux<AdvisedResponse> aroundStream(AdvisedRequest advisedRequest, AroundAdvisorChain chain);
```

Comment:
What's stopping the "Custom" implementations to be considered as the "around" advisors, making the "custom" abstraction unnecessary?